Marlin/Configuration_adv.h

```diff
@@ -289,7 +289,9 @@
   // You can get round this by connecting a push button or single throw switch to the pin defined as SDCARDCARDDETECT
   // in the pins.h file.  When using a push button pulling the pin to ground this will need inverted.  This setting should
   // be commented out otherwise
-  #define SDCARDDETECTINVERTED
+  #ifndef ELB_FULL_GRAPHIC_CONTROLLER
+    #define SDCARDDETECTINVERTED
+  #endif
```

Comment:
Instead of this being here, it should be in `Conditionals.h` as the opposite case:
```
#ifdef ELB_FULL_GRAPHIC_CONTROLLER
  #undef SDCARDDETECTINVERTED
#endif
```